src/_locales/en-US.json

```diff
@@ -78,6 +78,8 @@
 
   "settings-option-min-to-tray": "Minimize to Tray",
 
+  "settings-option-start-on-boot": "Start automatically on Bootup.",
```

Comment:
I feel as though this string should be something more like

"Automatically launch when I turn my computer on"